src/main/java/net/sf/jabref/gui/maintable/MainTable.java

```diff
@@ -194,6 +197,24 @@ public void actionPerformed(ActionEvent e) {
                 panel.selectPreviousEntry();
             }
         });
+
+        String selectFirst = "selectFirst";
+        im.put(Globals.getKeyPrefs().getKey(KeyBinding.SELECT_FIRST_ENTRY), selectFirst);
+        am.put(selectFirst, new AbstractAction() {
```

Comment:
I know that this screams lambda, but AbstracAction unfortunately does not have an functional Interface... :disappointed: 
Got fooled by it already ;)
http://blog.codefx.org/techniques/getting-rid-of-anonymous-classes/